src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1056,29 +1056,43 @@ PurgeStatement PurgeStatement(): {
 
 DescribeStatement Describe(): {
     Table table;
+    DescribeStatement stmt = new DescribeStatement();
```

Comment:
Next time, create the statement at the end only after all tokens have parsed and passed, like: 

```
return new Statement().withX(token.image).withY(token.image);

```

Its not a hard requirement, but in general Objects should be initialized only when needed.